theorems/T000080.md

```diff
@@ -1,14 +1,18 @@
 ---
 uid: T000080
 if:
-  P000009: true
+  and:
+  - P000009: true
+  - P000129: false
 then:
   P000060: false
 refs:
 - doi: 10.1007/978-1-4612-6290-9
   name: Counterexamples in Topology
 ---
 
-If $X$ is Urysohn and $x,y \in X$ are distinct, there is an $f:X \rightarrow \mathbb{R}$ with $f(x) \neq f(y)$.
+If $X$ is non-trivial, choose distinct $x,y \in X$, and
```

Comment:
It feels that the hypothesis of being "trivial" (= indiscrete topology) is the wrong property to use as a hypothesis here.  We cannot say "If X is non-trivial, choose distinct points x and y ..." (as exemplified by the singleton space, which has indiscrete topology, but does not have two different elements).

I think a more natural hypothesis here would be "the space contains at least two point".  This would be similar to other cardinality properties already in the database (Finite, Countable, Smaller than the continuum, etc), instead of a separation property like the space being indiscrete.

Reply:
I agree with you on this point. This needs to be looked at.

Reply:
@StevenClontz Would this be appropriate to address in the context of a separate issue/PR?

Reply:
For clarity, non-trivial implies plural cardinality, so the singleton isn't a counterexample. So this PR is correct, but it nonetheless could be improved as discussed. I'm going to make those improvements now and push the updates.